src/Handler/CurlMultiHandler.php

```diff
@@ -49,25 +68,34 @@ public function __construct(array $options = [])
             $this->selectTimeout = 1;
         }
 
-        $this->options = isset($options['options']) ? $options['options'] : [];
+        $this->options = $options['options'] ?? [];
     }
 
+    /**
+     * @param string $name
+     *
+     * @return resource
+     */
```

Comment:
Missing throws doc? The other `__get` lower down the diff has throws doc.

Reply:
Fixed in 9ef1da2e03cbbe60b29e23b4114b08f12cd8b5b0